docs/css/main.css

```diff
@@ -44,6 +44,16 @@ ul, li {
   }
 }
 
+@media screen and (min-width: 998px){
+  body {
+    padding: 0;
+  }
+}
+
+*, *:after, *:before {
```

Comment:
Can we go with `*::after`, `*::before*`? Single colon is only legacy support for IE8. :)

Reply:
you're right ;) also with reduntant declarations. thx